### PR TITLE
Capstone project final review

### PR DESCRIPTION
**Pull request by batoolfatima2135**
In this branch I have built these 2 pages:
1. The home page.
2. The About page.
Each of these pages is responsive and I have used Bootstrap for responsiveness

**Links**
The home page has a link to the About page in the menu.
The logo in the header links to the home page.
**Mobile menu**
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
The mobile menu has a close (X) button that closes the menu.
**Dynamic page**
The section "Featured courses" is created dynamically in JavaScript.
**See More button implementation is not necessary, it is an optional requirement.**

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            <li class="nav-item mx-3">
+              <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+            </li>
+            <li class="nav-item mx-3">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+  </nav>
```
Comment (V-Blaze) [inline index.html]: - [ ] You have done well implementing the `nav` bar and making it responsive, :+1: :+1:  But following the project requirement, I think there are some changes you need to implement here.
- - According to the project requirement, you are meant to replicate the given design as much as you can, Currently, your navbar doesn't have the top dark bar. and also the social links.
- - Also your main nav par is missing some links like the `program, sponsor, cc campaign` etc.. I'll suggest you go through the design one more time and add these :+1: :smile: 

- - Your Design:
Screenshot from 2023-05-23 14-48-51

- - PRovided design:

Screenshot from 2023-05-23 14-49-30

File: index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+ <div class="col-md-6 col-lg-6 col-10 text-center p-5 ">
+ <a href="#" class="text-light d-none d-md-block d-lg-block">SEE THE WHOLE COURSE</a>
+ <button class="d-block d-md-none d-lg-none mx-auto btn apply text-light p-3">Apply Now in techrotics</button>
+ </div>
+ </div>
+</section> 
+<section class="Featured-courses ">
+  <div class="row justify-content-center ">
+    <div class="col-11 d-flex flex-column align-items-center mt-5">
+      <h2 class="text-center text-dark ">Featured courses</h2>
```
Comment (V-Blaze) [inline index.html:145]: - [ ] Looking at the mobile version of the `Featured section` from the given design, There is a **`more button`** below the speakers and only two speakers are displayed, as you are meant to replicate the given design, I'll suggest you implement this in your own project as its part of the requirement. :+1: :100: :smile: 
- - Screenshot for reference:
Screenshot from 2023-05-23 15-05-03

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @batoolfatima2135  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Ui looks great ✔
-  Pull request summary is descriptive👋 
-  Very clean and organized code✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+                    <a class="navbar-brand " href="index.html"><img src="images/logo.png" height="40px"
+                            class="mt-3 mx-2 mt-lg-0 mt-md-0" alt="logo"></a>
+                    <ul class="navbar-nav  mb-2 mb-lg-0 mx-2 mt-md-2 mt-lg-1 ms-auto">
+                        <li class="nav-item mx-3">
+                            <a class="nav-link " aria-current="page" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item mx-3">
+                            <a class="nav-link active" href="about.html">About</a>
+                        </li>
```
Comment (mukhammadrizooff) [inline about.html]: - [ ] [REQUIRED] Good job so far! I appreciate what you have done it is so far kindly implement the [previous reviewer's suggestion](https://github.com/batoolfatima2135/Capstone/pull/1#discussion_r1202385795) also `ABOUT` page 🙏🏼 

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
```
Comment (mukhammadrizooff) [inline index.html:145]: - [ ] [REQUIRED] Good job so far! Please implement the [previous reviewer's suggestion](https://github.com/batoolfatima2135/Capstone/pull/1/files#r1202403616) to `speakers` section's mobile design 🙏🏼 

image

File: index.html (hunk 2 above)
Comment (Olamarx) [reply index.html:145]: > * [ ]  Looking at the mobile version of the `Featured section` from the given design, There is a **`more button`** below the speakers and only two speakers are displayed, as you are meant to replicate the given design, I'll suggest you implement this in your own project as its part of the requirement. 👍 💯 😄
> * * Screenshot for reference:
>     Screenshot from 2023-05-23 15-05-03

[OPTIONAL] I would implore you to implement this request to give your UI a good interaction there.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+        <div class="row justify-content-center ">
+            <div class="col-lg-3 col-md-4 col-6 align-self-center ">
+                <img src="images/logo.png" class="img-fluid" alt="logo">
+            </div>
+            <div class="col-5  align-self-center">
+                <p class="small">With more than 5,800 courses, Professional Certificates, and degrees from top-tier
+                    colleges and organisations, you may launch, change, or enhance your profession.</p>
+            </div>
+        </div>
+    </footer>
```
Comment (mukhammadrizooff) [inline about.html]: - [ ] [OPTIONAL] Good job so far! Please give the correct `BG` color to the `ABOUT` page's footer part and remove the `partners` part for this page 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @batoolfatima2135, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 
- Descriptive commit history ✔️ 
- Descriptive `README.md` file ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @batoolfatima2135, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Requested changes fixed ✔️
- GitHub flow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**